exercises/practice/resistor-color-duo/runtests.jl

```diff
@@ -0,0 +1,33 @@
+using Test
+
+include("resistor-color-duo.jl")
+
+@testset verbose = true "Resistor Color Duo" begin
+    @testset "Brown and black" begin
+        @test colorcode(["brown", "black"]) == 10
```

Comment:
It's still weird that this and the trio exercise pass in a vector rather than separate arguments. Is this related to the learning objectives for the exercise?

Reply:
As far as I know, that was by design. Notably, Haskell does separate arguments (see https://github.com/exercism/haskell/blob/main/exercises/practice/resistor-color-duo/src/ResistorColors.hs and https://github.com/exercism/haskell/blob/main/exercises/practice/resistor-color-trio/src/ResistorColors.hs). However since we already use a vector in resistor-color-trio, it makes sense to do the same here for consistency.

Reply:
Sure, maybe it'll help people who haven't worked out how to index collections yet or something.

Reply:
Should resistor-color-trio be updated then? You'll break solutions, but the whole resistor color series will be consistent on how they handle arguments.

Reply:
No, I don't think so. I was saying "sure let's do the same as trio for consistency".